src/mechanics/camera.rs

```diff
@@ -0,0 +1,14 @@
+use crate::Player;
+use bevy::prelude::*;
+
+pub fn move_camera(
+    player_query: Query<&Transform, (With<Player>, Changed<Transform>)>,
+    mut camera_query: Query<&mut Transform, (With<Camera>, Without<Player>)>,
+) {
+    if !player_query.is_empty() {
+        let player_transform = player_query.single();
+        let mut camera_transform = camera_query.single_mut();
+
+        camera_transform.translation = player_transform.translation;
+    }
```

Comment:
There's an opportunity here to use a neat Rust trick.

If we're treating this like a null/empty check like in other languages, we can ask for the queries to return something called a Result type. A Result type either contains what we want on success (Ok), or an Error with a reason of how otherwise.

If we don't care about the errors, then we can extract the value only when we get an Ok back, like this:
```rust
if let Ok(player_transform) = player_query.get_single() {
    if let Ok(mut camera_transform) = camera_query.get_single_mut() {
        camera_transform.translation = player_transform.translation;
    }
}
```